numpy/core/numeric.py

```diff
@@ -673,7 +673,7 @@ def correlate(a, v, mode='valid'):
         is 'valid', unlike `convolve`, which uses 'full'.
     old_behavior : bool
         `old_behavior` was removed in NumPy 1.10. If you need the old
-        behavior, use `multiarray.correlate`.
+        behavior, use ``multiarray.correlate``.
```

Comment:
this should be `correlate` (it's in the main numpy namespace, and `multiarray` is private)

Reply:
yes, please remove these lines

Reply:
Done.